Epsor/consumer-wrapper
# feat: 🎸 Add health check to consumer-wrapper

#### Technical checklist

- [x] I tested my code following testing cases.
- [x] I covered my code with automatic tests.
- [x] I added logs in case of error or business logic.
- [x] I well designed my commits.
- [x] I read my Pull Request's changes.
- [x] I respected provided design and wording.

---

How to test:

`cd consumer-wrapper && npm start`

go to [localhost:8000/.well-known/express/server-health](http://localhost:8000/.well-known/express/server-health)

expected output: `{"status":"pass"}`
